apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/global-style-sidebar-notice.tsx

```diff
@@ -0,0 +1,55 @@
+declare const wpcomGlobalStyles: { upgradeUrl: string; blogId: string };
+
+import { ExternalLink, Fill, Notice } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { createInterpolateElement, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { recordUpgradeNoticeSidebarShow, recordUpgradeSidebarNoticeClick } from './tracks-events';
+import { useGlobalStylesConfig } from './use-global-styles-config';
+
+const GLOBAL_STYLES_SIDEBAR = 'edit-site/global-styles';
+
+export function GlobalStylesSidebarNotice() {
+	const area = useSelect(
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
+		( select ) => select( 'core/interface' ).getActiveComplementaryArea( 'core/edit-site' ),
+		[]
+	);
+
+	const isGlobalStylesSidebar = GLOBAL_STYLES_SIDEBAR === area;
+
+	const globalStylesInUse = useGlobalStylesConfig().globalStylesInUse;
+
+	useEffect( () => {
+		if ( globalStylesInUse && isGlobalStylesSidebar ) {
+			recordUpgradeNoticeSidebarShow();
+		}
+	}, [ globalStylesInUse, isGlobalStylesSidebar ] );
+
+	return (
+		<Fill name="ComplementaryArea/core/edit-site">
+			{ /*
+			We'll need to do the condition here because if we are doing an early return, the fill will be introduced at the bottom of the page, which means some additional CSS magic needs to be done.
+			*/ }
+			{ globalStylesInUse && isGlobalStylesSidebar && (
+				<div className="interface-complementary-area">
+					<Notice status="warning" isDismissible={ false } className="wpcom-global-styles-notice">
+						{ createInterpolateElement(
+							__( '<a>Upgrade your plan</a> to keep your style changes.', 'full-site-editing' ),
```

Comment:
I'm unsure about saying "keep" here, since the styles are never lost – they are just not visible on the frontend, so they are technically kept even if you don't upgrade. 

Could we maybe reuse the same wording used in the other notice to say something like `Your changes include customized styles that will only be visible once you <a>upgrade to a Premium plan</a>`?